libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -109,6 +109,13 @@ export class MessageRepository extends BaseRepository<MessageEntity> {
     );
   }
 
+  async markAllUnseenAsSeen(subscriberId: string) {
+    return this.update(
+      { _subscriberId: subscriberId, seen: false },
+      { $set: { seen: true, lastSeenDate: new Date() } }
```

Comment:
I think a `$multi` flag is missing here? or is it taken care by the base repostiroy?

Reply:
Yes is taken care by base repository